test/e2e/clients/python/test_e2e_python_sdk.py

```
@@ -246,3 +246,82 @@ def test_python_sdk_gateway_mode_warmpool(
+        assert pvc_res is not None, f"PVC {pvc_name} should exist"
+        assert pvc_res.spec.resources.requests is not None
+        assert pvc_res.spec.resources.requests.get("storage") == "1Gi", "PVC storage request mismatch in cluster"
+        if storage_class:
+            assert pvc_res.spec.storage_class_name == storage_class, "PVC storageClassName mismatch in cluster"
+
+        print("Running command to verify sandbox is operational...")
+        res = sandbox.commands.run("df -h")
+        print(f"Disk space output:\n{res.stdout}")
+        assert res.exit_code == 0, f"Command df -h failed with exit code {res.exit_code}: {res.stderr}"
+
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)